client/app/src/accounts/view/addDelegate.html

```diff
@@ -23,10 +23,17 @@ <h2 translate>Add Delegate </h2>
           </md-card-title>
         </md-card> -->
 
-        <md-input-container class="md-block">
+        <md-input-container class="md-block" ng-if="!addDelegateDialog.data.registeredDelegates[0]">
           <label translate>Delegate name</label>
           <input ng-model="addDelegateDialog.data.delegatename" type="text">
         </md-input-container>
+        
+        <md-input-container class="md-block" ng-if="addDelegateDialog.data.registeredDelegates[0]">
+          <label for="delegate-name">Select delegate: </label>
+          <md-select id="delegate-name" ng-model="addDelegateDialog.data.delegatename">
+            <md-option ng-value="delegate.username" ng-repeat="delegate in addDelegateDialog.data.registeredDelegates | orderBy: 'username'">{{delegate.username}}</md-option>
```

Comment:
It SHOULD leave the possibility to override with a custom names, reasons:
- delegates not in 51 active
- new registered delegates, but the list has not been refreshed (needs to restart app to get fresh new list)